rancher/v1.2/en/quick-start-guide/index.md

```diff
@@ -31,12 +31,12 @@ If you are interested in trying one of our latest development builds which will
 All you need is one command to launch Rancher server. After launching the container, we'll tail the logs to see when the server is up and running.
 
 ```bash
-$ sudo docker run -d --restart=always -p 8080:8080 rancher/server
+$ sudo docker run -d --restart=always --name rancher-server -p 8080:8080 rancher/server
```

Comment:
can you change it to <containerID>? And if you want, you can add details on how to find the container ID. it shows the container ID after you run the `docker run` command.